### PR TITLE
pin scipy to max 1.6.2

### PR DESCRIPTION
**Pull request by esc**
As title

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,9 +17,10 @@
         "License :: OSI Approved :: BSD License",
         "Operating System :: OS Independent",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.0",
```
Comment (stuartarchibald) [inline setup.py:23]:
```suggestion
        "Programming Language :: Python :: 3.9",
```
?

Comment (esc) [reply setup.py:23]: yes, typo

Comment (esc) [reply setup.py]: fixed

**Review by stuartarchibald — APPROVED** [review]
Thanks for the patch.

File: setup.py (hunk 1 above)
Comment (stuartarchibald) [reply setup.py]: thanks

**Pull request merged**